src/components/shared/logo.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { FC } from 'react';
+import Anchor from 'components/anchor';
+import { css, SerializedStyles } from '@emotion/core';
+import { Logo } from 'capi';
+import { Format } from '@guardian/types/Format';
+import { darkModeCss } from 'styles';
+import { remSpace, text, neutral } from '@guardian/src-foundations';
+import { Branding } from '@guardian/apps-rendering-api-models/branding';
+import { Item, getFormat } from 'item';
+import { pipe2 } from 'lib';
+import { map, withDefault } from 'types/option';
+import { textSans } from '@guardian/src-foundations/typography';
+
+interface Props {
+    branding: Branding;
+    format: Format;
+}
+
+const styles = (lightModeImage: string, darkModeImage?: string): SerializedStyles => css`
+    margin: ${remSpace[9]} 0;
+    ${textSans.small()}
+
+    img {
+        content: url("${lightModeImage}");
```

Comment:
What was the reason behind using `content` in the CSS over `src` in the `<img>` tag out of curiosity? 🧐

Reply:
I wanted to make use of `darkModeCss` to use the alternative logo. I guess this could be done with client side JavaScript but I like having it in css!

They are `static.theguardian.com` urls so maybe? They don't seem to be the best of quality and they can contain random white space which puts our alignment off..